vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
+* Dropped support for suffix-mode installation. Only single-hostname is supported now.
+* Temporarily dropped support for cluster-level operator installation.
+* Removed EntandoClusterInfrastructure and EntandoCompositeApp CR.
+* k8s-service is now part of the base Entando infrastructure and therefore installed alongside the operator.
+* ComponentManager now communicates with k8s-service using service-to-service authentication, and strictly within the cluster network.
+* Entando Operator deployment is now simply called entando-operator.
+* k8s-service deployment dropped the application qualifier and is now simply called entando-k8s-service.
+* Fixed an intermittent issue with the operator that prevented installing new bundles until the operator was restarted.
+* Updated all the Entando base images for reliability and security.
+* Replaced the centos images to rocky images and rebased the derived images.
+* Updated the auto provisioned PostgreSQL image to version 14.
+* MySQL and PostgreSQL are now served by customized and updated entando-mysql and entando-postgres images.
+
+### Entando Bundler
 
-Entando implements the SQL versioning tool Liquibase to track, manage and deploy updates to databases. Refer to [Liquibase Migration](../reference/databases.md#liquibase-migration) to learn how the App Engine automates database schema management and modifications.
+* It’s now possible to extract protected assets.
+* Fixed several bugs in the extraction of pages and widgets.
+* Added support for categories.
+* Error handling has been improved.
+* It's now possible to generate the so-called system-level bundles (see ECR).
+* Extract/download optimized to only touch the relevant resources.
+* Libraries updated for reliability and security.
+
+### EOL Kubernetes Versions
+
+Entando7 drops official support for for OS3.11 and Kubernetes versions prior to 1.20.
+
+### Google Kubernetes Engine (GKE)
+
+* Postponed support to 7.0.1.
+
+### Azure Kubernetes Service (AKS)
+
+* Postponed support to 7.0.1.
+
+## Breaking Changes
+
+* The internal PostgreSQL version has been updated to version 14. Bundles that rely on the auto provisioning of PostgreSQL may (or may not) be affected.
+
+## Known Issues
+
+### ECG
+
+* The default "details" widget generated for an entity doesn't work out of the box.
+* Local Keycloak loses state across restarts when using the `--no-recreate` option.
+
+### ECR
+
+* An in-place upgrade from version 6.3.X to version 7 of the ECR is not possible, but a simple migration script will be provided. In-place upgrades from version 7 to future versions will be possible.
+
+### GKE
+
+* There is still a known issue with the current NGINX ingress that can be worked around. The issue will be solved in the next minor or patch release (6.4.0 or 6.3.3). 
+
+## Deprecation Warnings
+
+* Support for versions 1 and 2 of the plugin descriptor are deprecated and will be removed in the future.
```

Comment:
@w-caffiero-entando we don't have any docs on versions 1 and 2. Is there a summary somewhere?  I'm very sure version 1 is completely broken several versions ago since using it even on 6.3.2 was not an option as a workaround for the environment variable problem and it worked with version 1.  I'm tempted to just remove this line.

Reply:
I think this was a leftover from 6.3.2 and this line should be removed now.